client-flink/flink-1.14/src/test/java/org/apache/celeborn/plugin/flink/RemoteShuffleMasterSuiteJ.java

```diff
@@ -64,6 +64,7 @@ public void setUp() {
     int startPort = Utils$.MODULE$.selectRandomInt(1024, 65535);
     configuration.setInteger("celeborn.master.port", startPort);
     configuration.setString("celeborn.master.endpoints", "localhost:" + startPort);
+    configuration.setString("celeborn.client.application.heartbeatInterval", "60s");
```

Comment:
Could this update to 30s? IMO, 30s is enough.

Reply:
done